app/src/main/java/org/simple/clinic/deeplink/DeepLinkUpdate.kt

```diff
@@ -18,18 +18,19 @@ class DeepLinkUpdate : Update<DeepLinkModel, DeepLinkEvent, DeepLinkEffect> {
       model: DeepLinkModel,
       event: UserFetched
   ): Next<DeepLinkModel, DeepLinkEffect> {
-    return when {
-      event.user?.loggedInStatus == User.LoggedInStatus.LOGGED_IN -> {
-        val effect = if (model.patientUuid != null) {
+    return if (event.user?.loggedInStatus == User.LoggedInStatus.LOGGED_IN) {
+      val effect = if (event.user.capabilities == User.Capabilities(User.CapabilityStatus.Yes)) {
```

Comment:
Can we extract these boolean checks into computed properties on the `User` model?

This will make this easier to understand, like `event.user?.isUserLoggedIn` and `event.user?.capabilities?.canRecordTeleconsultLog`.

Reply:
I don't see this change in branch. Was it not pushed? 🤔

Reply:
Sorry, I think I did the capabilities part and missed this. I will re-check from now as well.